app/assets/javascripts/users.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```

Comment:
@jw9billions not being used, please delete. Remember to only commit files you are using :)